io.github.dfandrich.gpscorrelate.json

```diff
@@ -0,0 +1,63 @@
+{
+  "app-id":"io.github.dfandrich.gpscorrelate",
+  "runtime":"org.freedesktop.Platform",
+  "runtime-version":"20.08",
+  "sdk":"org.freedesktop.Sdk",
+  "command":"gpscorrelate-gui",
+  "finish-args":[
+    "--socket=fallback-x11",
+    "--socket=wayland",
+	"--filesystem=home",
+    /* Needed for gvfs to work */
+    "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*"
+  ],
+  "modules":[
+   {
+            "name": "exiv2",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DEXIV2_ENABLE_LENSDATA=off",
+                "-DEXIV2_BUILD_SAMPLES=off",
+                "-DEXIV2_ENABLE_LENSDATA=off"
+            ],
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/lib/cmake",
+                "/lib/pkgconfig",
+                "/lib/*.a",
+                "/share/man"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.exiv2.org/builds/exiv2-0.27.3-Source.tar.gz",
+                    "sha256": "a79f5613812aa21755d578a297874fb59a85101e793edc64ec2c6bd994e3e778"
+                }]
+    },
+	{
```

Comment:
The whole modules seems to be mis-indented

Reply:
Fixed. There were a few tabs hidden in there, too.